src/functions-and-arrays.js

```diff
@@ -52,14 +98,32 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+let newArray=[]
+
+for (let i=0; i<arr.length; i++){
+  if (newArray.indexOf(arr[i]) === -1 ){
+    newArray.push(arr[i])
+  }
+}
+return newArray
+ }
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(arr, word) { 
+ for (let i=0; i<arr.length; i++){
+    if (arr[i] === word){
+      return true
+    } else {
+      return false
+    }
+  }
+  return arr, word
+}
```

Comment:
- Aquí, tu función lo que hace es que en cada función comprueba si la palabra sobre la que está iterando y la palabra introducida como `word` son la misma. Si es así, devuelve `true` (lo cual está bien), pero si no es, devuelve `false`, y sale del bucle, por lo tanto saldría del `for` solo con que la `word` no estuviera la primera en el array.
- Luego la función devuelve el array entero seguido de la palabra introducida como segundo parámetro, pero el ejercicio te pide que simplemente te devuelva `true` si la palabra está o `false` si no.
- Esto lo podrías solucionar haciendo que la función devuelva true si encuentra la coincidencia (esto dentro del bucle), y si acaba el bucle pero no ha encontrado ninguna coincidencia y no ha hecho `return`, devuelva `false`.
```suggestion
function doesWordExist(arr, word) {
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] === word) {
      return true
    }
  }
  return false
}
```